app/src/main/res/values-de/strings.xml

```diff
@@ -129,8 +129,8 @@
     <string name="enable_in_home">Zu Hause aktivieren</string>
     <string name="enable_in_search">In der Suche aktivieren</string>
     <string name="check">Überprüfen</string>
-    <string name="buy_text">Grayjay ist weder eine einfache noch eine günstige App zum Aufbauen und Warten. Wir haben Vollzeit-Ingenieure, die an der App und den umliegenden Systemen arbeiten. Und wird wahrscheinlich nicht so bald, wenn überhaupt, sein Geld zurückverdienen.\n\nDie Mission von FUTO besteht darin, dass Open-Source-Software und nicht böswillige Geschäftspraktiken für Projekte und ihre Entwickler eine nachhaltige Einnahmequelle werden. Aus diesem Grund sind wir dafür, dass die Benutzer tatsächlich für die Software bezahlen.\n\nAus diesem Grund möchte Grayjay, dass Sie für die Software bezahlen.</string>
-    <string name="an_uncaught_exception_was_thrown_we_re_sorry_for_the_inconvenience">Ein ungefangene Ausnahme wurde ausgelöst, es tut uns leid für die Unannehmlichkeiten.</string>
+    <string name="buy_text">Es ist weder einfache noch günstige, eine App wie Grayjay, zu Entwikeln und zu Warten. Wir beschäftigen Vollzeit-Ingenieure, die an der App und den umliegenden Systemen arbeiten. Grayjay wird wahrscheinlich erst spät, wenn überhaupt, Geld zurückverdienen.\n\nDie Mission von FUTO besteht darin, dass Open-Source-Software und nicht böswillige Geschäftspraktiken für Projekte und ihre Entwickler eine nachhaltige Einnahmequelle werden. Aus diesem Grund sind wir dafür, dass die Benutzer tatsächlich für eine solche Software bezahlen.\n\nAus diesem Grund möchte Grayjay, dass Sie für die Software bezahlen.</string>
```

Comment:
I found some minor issues with your new text (missspelling and so on) and fixed it in this comment.

Es ist weder einfach noch günstig, eine App wie Grayjay zu entwickeln und zu warten. Wir beschäftigen Vollzeit-Ingenieure, die an der App und den umliegenden Systemen arbeiten. Grayjay wird wahrscheinlich erst spät, wenn überhaupt, Geld zurückverdienen.

Die Mission von FUTO besteht darin, dass Open-Source-Software und nicht böswillige Geschäftspraktiken für Projekte und ihre Entwickler eine nachhaltige Einnahmequelle werden. Aus diesem Grund sind wir dafür, dass die Benutzer tatsächlich für solche Software bezahlen.

Aus diesem Grund möchte Grayjay, dass Sie für die Software bezahlen.